build/android/make-toolchain

```diff
@@ -1,13 +1,20 @@
 #!/bin/sh
 set -e
 
-if [ $# -ne 3 ]; then
-    echo "Usage: $0 NDK_DIR ARCH API" 1>&2
+if [ $# -ne 2 -a $# -ne 3 ]; then
+    echo "" 1>&2
+    echo "Usage: $0 NDK_DIR ARCH [API]" 1>&2
+    echo "" 1>&2
     echo "  NDK_DIR: path where NDK is installed" 1>&2
-    echo "  ARCH: aarch64-linux-android arm-linux-androideabi mipsel-linux-android mips64el-linux-android x86 x86_64" 1>&2
-    echo "  API: 1 ... 21" 1>&2
-    echo "Example usage (on MacOS using brew):" 1>&2
-    echo "  $0 /usr/local/Cellar/android-ndk/r10e aarch64-linux-android 21" 1>&2
+    echo "  ARCH: armeabi, armeabi-v7a, arm64, mips, mips64, x86, x86_64" 1>&2
+    echo "  API: 1 ... 25" 1>&2
+    echo "" 1>&2
+    echo "If API is omitted, the minimum possible API for the " 1>&2
+    echo "required architecture is used." 1>&2
+    echo "" 1>&2
+    echo "Example usage (on MacOS with Android Studio installed):" 1>&2
+    echo "  $0 ~/Library/Android/sdk/ndk-bundle arm 19" 1>&2
+    echo "" 1>&2
     exit 1
```

Comment:
Now API is optional. If not specified we use the lowest possible API for architecture.